pkg/dashboard/ui/package.json

```diff
@@ -48,7 +48,7 @@
     "i18next-chained-backend": "^3.0.2",
     "i18next-localstorage-backend": "^3.1.3",
     "i18next-xhr-backend": "^3.2.2",
-    "iguazio.dashboard-controls": "1.0.12-1.13.x",
+    "iguazio.dashboard-controls": "^1.0.13-1.13.x",
```

Comment:
@illia-prokopchuk CI fails with:
```
462.9 .date-time-picker-color-set is undefined in file node_modules/iguazio.dashboard-controls/dist/less/iguazio.dashboard-controls.less line no. 6040
```

IIRC, this is why we removed the `^` from the version (see https://github.com/nuclio/nuclio/pull/3446)
Remove it from this PR as well.

Reply:
@TomerShor done